_includes/guide/try_the_api.html

```diff
@@ -85,10 +86,10 @@ <h2>
 	Step 2: Start a Job
 </h2>
 <p>
-	With our new access token, we are clear to make more requests to the API. We will now tell the server to serve up our data.
+	With our new access token, we are able to make more requests to the API. We will now tell the server to deliver our data.
 </p>
 <p>
-	In this example, we're requesting data from the /Patient endpoint with no date filter (no _since parameter) and no specified resource type (no _type parameter). We will include the long string of characters from the previous step (our access token) in the authorization header (-H "Authorization: Bearer "). Notice that we need to include the word Bearer with a space before the access token.
+	In this example, we requested data from the /Patient endpoint with no date filter (no _since parameter) and no specified resource type (no _type parameter). We included the long string of characters from the previous step (our access token) in the authorization header (-H "Authorization: Bearer "). Notice that we needed to include the word "Bearer" followed by a space before the access token. We also added "-v" in order to see the ob ID.
```

Comment:
Typo, should state "We also added "-v" in order to see the **Job ID**" not "We also added "-v" in order to see the ob ID"